Certificates/cert-gen.py

```diff
@@ -0,0 +1,106 @@
+# Requires inkscape in path and PDF module
+import argparse
+import csv
+import numpy as np
+import os
+import pathlib
+import sys
+from PDF import PdfFileReader, PdfFileWriter
+from subprocess import Popen
+
+def argument_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        'template_svg',
+        help="Input svg template with text matching 'NAME OF STUDENT IN CAPS' and 'NAME OF SCHOOL IN CAPS' for "
+             "replacement with the students' names and schools'/teams' names, respectively, "
+             "on the produced certificates.",
+        type=pathlib.Path,
+    )
+    parser.add_argument(
+        'input_data',
+        help="Input csv with columns of 'First Name', 'Last Name', and 'School'."
+             "In absence of this argument, the script's own test data will be used.",
+        type=pathlib.Path,
+        nargs='?',
+        default=-1,
+    )
+    parser.add_argument(
+        'output_path',
+        help="Path in which the output will be deposited. Defaults to current working directory.",
+        type=pathlib.Path,
+        nargs='?',
+        default=os.getcwd(),
+    )
+    return parser
+
+def main(arguments):
+    options = argument_parser().parse_args(arguments)
```

Comment:
you don't need the `arguments` variable here, `parse_args()` works just fine

Reply:
Aha :) should've read the docs ;)